netlify.toml

```diff
@@ -31,7 +31,7 @@ status = 302
 # Special redirect after introduction of Bonita branding version
 # Add a new entry when adding a new version/branch. This is generally done when the developments of a new version start
 [[redirects]]
-from = "/bonita/7.16/*"
+from = "/bonita/8.0/*"
```

Comment:
I suggest we keep both the 7.16 and 8.0 redirects as 7.16 has been in the air for a while.
Redirects are free with Netlify, we should try to avoid http 404 errors as much as possible. 😸

Reply:
Fair enough !